src/app/core/utils/api-token/api-token.service.ts

```diff
@@ -258,6 +262,16 @@ export class ApiTokenService {
     return this.appendAuthentication(req).pipe(
       concatMap(request =>
         next.handle(request).pipe(
+          map(event => {
+            if (event instanceof HttpResponse && event.url.endsWith('-/token') && request.body instanceof HttpParams) {
+              // eslint-disable-next-line @typescript-eslint/naming-convention
+              const { id_token, ...body } = event.body;
```

Comment:
Could you not rename the destructured variable like this: `const { id_token: _, ...` and avoid the ESLint exception?

Reply:
Zu Befehl